recipe/meta.yaml

```
@@ -59,7 +59,7 @@ requirements:
     - macholib >=1.8
     - pyinstaller-hooks-contrib >=2020.6
     - setuptools  # due to pkg_resources import
-    - importlib-metadata  # [py<38]
+    - importlib-metadata >=0.8 # [py<38]
```

Is this what the linter wants?
```suggestion
    - importlib-metadata >=0.8  # [py<38]
```

---

Yes!

---

Please bump the build number too :)